### PR TITLE
Alerting WIP update documentation

### PR DESCRIPTION
**Pull request by davidmparrott**
**What this PR does / why we need it**:
Updates to the docs site for unified alerting

**Special notes for your reviewer**:
Will remove WIP but please read as updates come so that you can comment and suggest edits.

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
These are great changes. One question, are the updated content still relevant for the older alerting? Asking because this topic will serve as the landing page for the alerting topics and will have links to older and new alerts. Thanks!

**Comment by davidmparrott** [comment]
@achatterjee-grafana I'm writing this specifically for the unified alerting. I don't think the docs for the older dashboard alerts will need an update so perhaps we should just link back to those on the overview page? Or if you have a suggestion I will gladly take it. All of my knowledge of the existing system comes from my work to build the new one.

**Comment by achatterjee-grafana** [comment]
> @achatterjee-grafana I'm writing this specifically for the unified alerting. I don't think the docs for the older dashboard alerts will need an update so perhaps we should just link back to those on the overview page? Or if you have a suggestion I will gladly take it. All of my knowledge of the existing system comes from my work to build the new one.

Makes sense!

File: docs/sources/alerting/_index.md [hunk 1]
```diff
@@ -6,49 +6,47 @@ weight = 110
...
+Alerts allow you to know about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services.
 
-Alerts consists of two parts:
+Alerts have four main components:
 
-- Alert rules - When the alert is triggered. Alert rules are defined by one or more conditions that are regularly evaluated by Grafana.
-- Notification channel - How the alert is delivered. When the conditions of an alert rule are met, the Grafana notifies the channels configured for that alert.
-
-Currently only the graph panel visualization supports alerts.
+- Alert rule - One or more conditions, the frequency of evaluation, and the (optional) duration that a condition must be met before notifying.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/_index.md:13]:
```suggestion
- Alert rule - Consists of one or more conditions, the frequency of evaluation, and the (optional) duration that a condition must be met before notifying.
```

**Review by achatterjee-grafana — APPROVED** [review]
One minor copy edit suggestion, otherwise good to go!

File: docs/sources/alerting/_index.md (hunk 1 above)
Comment (domasx2) [inline docs/sources/alerting/_index.md:13]: should we make a distinction between alert rules and alerts (insances) perhaps?

File: docs/sources/alerting/_index.md [hunk 2]
```diff
@@ -6,49 +6,47 @@ weight = 110
...
 
 ## Alert tasks
 
 You can perform the following tasks for alerts:
 
-- [Add or edit an alert notification channel]({{< relref "notifications.md" >}})
 - [Create an alert rule]({{< relref "create-alerts.md" >}})
 - [View existing alert rules and their current state]({{< relref "view-alerts.md" >}})
 - [Test alert rules and troubleshoot]({{< relref "troubleshoot-alerts.md" >}})
+- [Add or edit an alert notification channel]({{< relref "notifications.md" >}})
```
Comment (domasx2) [inline docs/sources/alerting/_index.md:25]: no such thing as "notification channel" :) there are "contact points" and "notification policies"

File: docs/sources/alerting/_index.md (hunk 1 above)
Comment (davidmparrott) [reply docs/sources/alerting/_index.md:13]: The instances are the output of the alerting system, where as these are the configuration components that define the behavior of it. We will need to give plenty of love to the topic of alert instances (they will need their own page, I think) but I don't think here is the correct place for that.

File: docs/sources/alerting/_index.md (hunk 2 above)
Comment (davidmparrott) [reply docs/sources/alerting/_index.md:25]: Thanks!

**Pull request merged**